File: PetStoreClient/petStore/src/app/models/order.model.ts
export class Order{
    
    orderId: number | null;
    firstName: string | null;
    lastName: string | null;
    address: string | null;
    status: string | null;

    constructor(orderId?: number, firstName?: string, lastName?: string, address?: string, status?: string){
        this.orderId = orderId || null;
        this.firstName = firstName || null;
        this.lastName = lastName || null;
        this.address = address || null;
        this.status = status || null;
    }
}

File: PetStore/Models/OrderModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace PetStore.Models
{
    public class OrderModel
    {
        public int OrderId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public string Status { get; set; }
    }
}
